.travis.yml

```diff
@@ -24,3 +24,15 @@ script:
 
 after_success:
     coveralls
+
+deploy:
+  provider: pypi
+  distributions: sdist
+  user: CityOfZion
+  password:
+    secure: SECURE_PASSWORD_HERE
+  on:
+    tags: true
+    all_branches: true
```

Comment:
it should only do this on tags, not all branches. See this for reference https://github.com/CityOfZion/neo-python-core/blob/master/.travis.yml#L36

Reply:
In my experience adding only `tags: true` did not trigger the deploy.
I found an issue on travis from a few years ago where people reported to have the same issue.
I'm gonna try once again without it and see how it goes.

Reply:
the linked travis.yml file above works. please change this PR to use the exact same way as neo-python-core.